plugins/python-build/bin/python-build

```diff
@@ -2222,6 +2222,19 @@ if [[ "$PYTHON_CONFIGURE_OPTS" != *"--enable-unicode="* ]]; then
   fi
 fi
 
+# regex_to_match="(--with-tcltk-libs='([^']+)')"
+if [[ "$PYTHON_CONFIGURE_OPTS" =~ (--with-tcltk-libs=\'([^\']+)\') ]]; then
+  tcltk_match=${BASH_REMATCH[1]}
```

Comment:
Please add surrounding double quotes.

Reply:
im not sure I understand, double quotes around the regex? like `"(--with-tcltk-libs=\'([^\']+)\')"` ?   That wont work.  Ideally I guess I can make the regex a var that it matches so then you dont have the \ to escape the single quote.  Or are you referring to something else

Reply:
Oh, Im guessing you are meaning around `${BASH_REMATCH[1]}` and similar? will do